### DISCUSSION TITLE
How do I launch the program

### DISCUSSION THREAD

**Discussion by Quetzal1123** [tryashtar/nbt-studio#72] [Q&A]
As the title describes, my issue would be the fact that I can't launch it. Perhaps there was something I missed, but as far as I can tell, there is nothing in the description that details the lauching of the program. Perhaps users are expected to know the details already, (and also it doesn't help that I am technologically-challenged at times) but I can't figure out for the life of me how to launch the program.

Help would be greatly appreciated, thanks.

**Comment by Hippie459MN** [comment]
Do you have .NET Desktop Runtime 6.0 installed? If not you need to install that otherwise it wont run. It works just fine here for me. I just double click on it to open it. If you dont have .NET Desktop Runtime 6.0 installed, you can get it here - https://dotnet.microsoft.com/en-us/download/dotnet/6.0/runtime
